supabase/migrations/20230814093932_log_entries_rls.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE log_entries ENABLE ROW LEVEL SECURITY
```

Comment:
@devpanther Did you test how the bot works with this "restrict all" policy? As far as I understand this policy does not allow writing to a DB to any of the users.

Reply:
It allows writing if you're using a service role api key which the bot is using..

Reply:
> @devpanther Did you test how the bot works with this "restrict all" policy? As far as I understand this policy does not allow writing to a DB to any of the users.

It allows writing if you're using a service role api key which the bot is using..

Reply:
Yeah, @devpanther 's right here. we already switched using service role key in backend.